src/overlay/PeerManager.cpp

```diff
@@ -570,6 +570,60 @@ PeerManager::dropAll(Database& db)
     db.getSession() << kSQLCreateStatement;
 }
 
+std::vector<std::pair<PeerBareAddress, PeerRecord>>
+PeerManager::loadAllPeers()
+{
+    ZoneScoped;
+    std::vector<std::pair<PeerBareAddress, PeerRecord>> result;
+    std::string sql = "SELECT * FROM peers";
```

Comment:
avoid using `SELECT *` as it's a footgun, just list the column names that you expect

Reply:
good call, updated.